miden-lib/asm/miden/kernels/tx/memory.masm

```diff
@@ -120,6 +120,9 @@ const.ACCT_CORE_DATA_SECTION_END_OFFSET=404
 # The memory address at which the account storage slot type data beings
 const.ACCT_STORAGE_SLOT_TYPE_DATA_OFFSET=405
 
+# The memory address at which the account procedures section begins.
+const.ACCT_PROCEDURES_SECTION_OFFSET=700
```

Comment:
@bobbinth would you change the naming here ?

Reply:
I think the naming looks good.